docs/community/KPI-options.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
I think this should live in a subsection within our EMP. @smb2796 do you agree?

Reply:
Personally I'd like to avoid having multiple levels of nesting within the sidebar. I'd be fine with having it in the `Synthetic Tokens` section, but also think it works in community. Don't feel strongly one way or the other!